doc/related_projects.rst

```diff
@@ -12,6 +12,12 @@ Interoperability and framework enhancements
 These tools adapt scikit-learn for use with other technologies or otherwise
 enhance the functionality of scikit-learn's estimators.
 
+- `Machine Learning <https://github.com/jeff1evesque/machine-learning>`_ an automated build
```

Comment:
I think that's called "good engineering". It doesn't need to be mentioned here where we'd really like the description to be as concise but informative as possible.